index.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Hannah Task - Home</title>
+    <title>Hannah Gorton Website</title>
```

Comment:
Do you need to specify that it's a website.

```suggestion
    <title>Hannah Gorton</title>
```